<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nap Time Systems</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>170</width>
      <height>122</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCommandLinkButton" name="btnPlayer">
       <property name="text">
        <string>Player</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCommandLinkButton" name="btnMemos">
       <property name="text">
        <string>Memos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCommandLinkButton" name="btnIntercom">
       <property name="text">
        <string>Intercom</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionConfig"/>
    <addaction name="separator" />
    <addaction name="actionNfs"/>
    <addaction name="actionNapListener"/>
    <addaction name="separator" />
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuConfig"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConfig">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionNfs">
   <property name="text">
    <string>Remount /mnt/remote nfs share</string>
   </property>
  </action>
  <action name="actionNapListener">
   <property name="text">
    <string>Restart naplistener</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
